superflore/generators/ebuild/gen_packages.py

```diff
@@ -42,145 +44,86 @@
 
 no_python3 = ['tf']
 
-
-def warn(string):
-    print(colored('>>>> {0}'.format(string), 'yellow'))
-
-
-def ok(string):
-    print(colored('>>>> {0}'.format(string), 'green'))
-
-
-def err(string):
-    print(colored('!!!! {0}'.format(string), 'red'))
+org = "Open Source Robotics Foundation"
+org_license = "BSD"
 
 
-def make_dir(dirname):
+def regenerate_pkg(overlay, pkg, distro_name=None, distro=None, update=False):
```

Comment:
Do you really need both distro and distro_name support? That seems like it is just going to be more to test. Since it's only a one line command to get the distro. It would be better to do that from the calling instance and not have the ambiguous API.

Reply:
> Since it's only a one line command to get the distro. It would be better to do that from the calling instance and not have the ambiguous API.

Ya, that's true. I was going for an optimization here, but I don't think I really saved myself anything.